rest/taskrouter/statistics/workspace/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,35 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using Newtonsoft.Json.Linq;
+using System;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+
+        var stats = WorkspaceStatisticsResource.Fetch(workspaceSid);
+        var cumulativeStats = JObject.FromObject(stats.Cumulative);
+
+        Console.WriteLine(cumulativeStats["avg_task_acceptance_time"]
```

Comment:
You don't need to use the explicit `Value<T>()` method to cast the results into something else. Please take a look at this example https://dotnetfiddle.net/djiDPN and change your examples accordingly.